lib/review/pdfmaker.rb

```diff
@@ -471,6 +479,8 @@ def latex_config
         result << "%% END: config-local.tex.erb\n"
       end
       result
+    rescue StandardError => e
```

Comment:
```suggestion
    rescue ApplicationError => e
```

`ReVIEW::ConfigError`等を拾うのであれば `ApplicationError` でできます（何でもStandardErrorで握りつぶしてしまうのは良くなさそうなので）

Reply:
ERBのbindでエラーが出たときにSyntaxError, ZeroDivisionErrorなど予想がつかない生の例外がきてしまうんですよね。ApplicationErrorで拾えなくて困っています。